.github/workflows/build.yml

```diff
@@ -96,24 +18,25 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-            target
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-cargo-
-      - run: cargo build --release
-      - name: tar it up
+      - name: Build release
+        run: cargo build --release
+      - name: Archive as .tar.gz (Linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: tar cfz htmlq-x86_64-linux.tar.gz -C target/release htmlq
+      - name: Archive as .tar.gz (macOS)
+        if: matrix.os == 'macos-latest'
+        run: tar cfz htmlq-x86_64-darwin.tar.gz -C target/release htmlq
+      - name: Archive as .tar.gz (Windows)
+        if: matrix.os == 'windows-latest'
         shell: bash
-        run: |
-          tar cfz htmlq_x64_win32.tar.gz -C target/release htmlq
-      - name: upload windows release
-        id: release
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: tar cfz htmlq-x86_64-windows.tar.gz -C target/release htmlq.exe
```

Comment:
I think it would be better to bundle the Windows release as a zip file since Windows doesn't support tar files.

Reply:
Done.